basic_module/account/src/account/checktx_handler.rs

```diff
@@ -0,0 +1,48 @@
+// use super::view::{get_balance, get_sequence};
+use super::internal::get_storage_id;
+use crate::{get_context, Account, Action, Address, Transaction};
+
+pub trait CheckTxHandler {
```

Comment:
To comply with FML rules, we should make one explicit trait implementation rather than default implementation.